nyc_taxis/operations/default.json

```diff
@@ -960,14 +960,18 @@
       "name": "avg_passenger_count_esql_segment_partitioning",
       "operation-type": "esql",
       "query" : "FROM nyc_taxis | stats avg(passenger_count)",
+  {# non-serverless-doc-partitioning-marker-start #}{%- if build_flavor != "serverless" or serverless_operator == true -%},
```

Comment:
Would it not be easier to simply delete the entire body, since this is the default option?

Reply:
Done